### PR TITLE
docs: Comment about Crashlytics with Dart-only installation

### PR DESCRIPTION
**Pull request by artem-zaitsev**
## Description
Add comment about Dart-only installation

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ x] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: docs/crashlytics/overview.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ dependencies {
 }
 
 // This must appear at the bottom of the file
-apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.gms.google-services' // If you use Dart-only installation don't apply this line
```
Comment (russellwheatley) [inline docs/crashlytics/overview.mdx:57]:
```suggestion

```

Comment (russellwheatley) [reply docs/crashlytics/overview.mdx:57]: I think it should be removed now as the documentation context refers to dart initialisation. To use the previous initialisation method, users would use [these instructions](https://firebase.flutter.dev/docs/manual-installation/android) I believe.

**Review by russellwheatley — CHANGES_REQUESTED** [review]

File: docs/crashlytics/overview.mdx (hunk 1 above)
Comment (artem-zaitsev) [reply docs/crashlytics/overview.mdx:57]: yep, i think you are right

**Comment by Salakar** [comment]
The documentation here is correct, applying the google services plugin is required for the optional native platform integration here for Android, the first paragraph states:

> Dart-only Firebase initialization currently only supports reporting Dart exceptions. To report native Android & iOS exception please follow the Android platform Setup section from the **manual installation** guide and the steps below.

.. of which applying the Google services plugin is part of the manual installation steps.

**Pull request closed without merging**